frameworks/keyed/ember/app/templates/components/my-table.hbs

```diff
@@ -1,73 +1,72 @@
 <div class="jumbotron">
-    <div class="row">
-        <div class="col-md-6">
-            <h1>Ember v3.5.1</h1>
+  <div class="row">
+    <div class="col-md-6">
+      <h1>Ember v3.9.1</h1>
+    </div>
+    <div class="col-md-6">
+      <div class="row">
+        <div class="col-sm-6 smallpad">
+          <BsButton id="run" @onClick={{action this.create}}>
```

Comment:
I think convention now is to use `@action` in the js, rather than the in the template

Reply:
yeah, but `fn` non landed (